p/scripts/main.js

```diff
@@ -960,6 +960,17 @@ function init_stream(stream) {
 			return false;
 		}
 
+		el = ev.target.closest('.item.share > a[data-type="clipboard"]');
+		if (el) {	//Clipboard
+			const textarea = document.createElement('textarea');
+			textarea.textContent = el.href;
+			document.body.appendChild(textarea);
+			textarea.select();
+			document.execCommand('copy');
+			document.body.removeChild(textarea);
+			return false;
```

Comment:
You think https://developer.mozilla.org/en-US/docs/Web/API/Clipboard/writeText is too new?

Reply:
I didn't know about it. It seems that it's fine but it is not supported by Internet Explorer.
I was searching if we still support it but couldn't find that information in the doc. If we don't, I'll change that.

Reply:
I found the information and we still support IE.

Reply:
IE is dead now, cf. what I wrote in https://github.com/FreshRSS/FreshRSS/issues/2159#issuecomment-443409899

Reply:
Yes, I think it is OK not to have IE11 support for new features. It is better if it does not break/crash though, for now, but that can be tested with feature detection